kafka_utils/kafka_check/commands/min_isr.py

```diff
@@ -33,47 +34,63 @@ def build_subparser(self, subparsers):
                  'is no settings in Zookeeper for partition.',
         )
         subparser.add_argument(
-            '--default_min_isr',
+            '--default-min-isr',
             type=int,
             default=1,
             help='Default min.isr value for cases without settings in Zookeeper '
             'for some topics. Default: %(default)s',
         )
         return subparser
 
-    def get_min_isr(self, topic):
-        """Return the min-isr for topic, or None if not specified"""
-        ISR_CONF_NAME = 'min.insync.replicas'
-        try:
-            config = self.zk.get_topic_config(topic)
-        except NoNodeError:
-            return None
-
-        if ISR_CONF_NAME in config:
-            return int(config[ISR_CONF_NAME])
-        else:
-            return None
-
     def run_command(self):
         """Min_isr command, checks number of actual min-isr
         for each topic-partition with configuration for that topic."""
-        topics = self.zk.get_topics()
-        not_in_sync = 0
+        broker_list = self.zk.get_brokers()
```

Comment:
ClusterConfig.broker_list is already in a KafkaClient compatible format. I think that all the other tools just use the broker_list from ClusterConfig instead of reading and building the broker_list from zookeeper.